src/realm/object-store/set.cpp

```diff
@@ -72,6 +72,16 @@ size_t Set::find(const T& value) const
     return as<T>().find(value);
 }
 
+size_t Set::find(Query&& q) const
+{
+    verify_attached();
+    if (m_type == PropertyType::Object) {
```

Comment:
Seems like find should just specialised on LnkSet, no?

Reply:
This logic is copied straight from list

Reply:
Not sure that makes it right 😬